emulcomm.py

```diff
@@ -84,6 +84,10 @@
 allowediplist = []
 cachelock = threading.Lock()  # This allows only a single simultaneous cache update
 
+# Trying to send a UDP datagram larger than this size will generate uncatchable
+# exceptions on some systems (OS X). See repy_v2 issue 113.
+MAX_ALLOWABLE_DGRAM_SIZE = 9216
```

Comment:
I guess this is in bytes?

Reply:
Yes; comment adapted to mention this explicitly.